src/bioregistry/data/bioregistry.json

```diff
@@ -30588,6 +30588,35 @@
     "pattern": "^\\d{7}$",
     "uri_format": "http://purl.obolibrary.org/obo/ECAO_$1"
   },
+  "eccDNA": {
```

Comment:
For new prefixes, the convention is to use all lowercase, but if capitalization is important to preserve in some form, then it can be done via synonyms (see also https://github.com/biopragmatics/bioregistry/blob/main/docs/CONTRIBUTING.md#minimum-new-prefix-requirements)